client/src/components/CtaButton.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+export default function CtaButton({ buttonText }) {
```

Comment:
Do you plan to have another Button related component which will not be a CTA button? Otherwise you can consider to rename it to just <Button /> so you can use that component even more flexible (to also create non-cta buttons)